src/components/events/Events.js

```diff
@@ -200,30 +228,89 @@ const Events = () => {
           onButtonClick={loadGetEvents}
           onResetSearch={handleRefreshSearch}
         />
+        <br />
+        <Typography
+          style={{ color: "black", mb: 2, fontSize: 14, align: 'right' }}
+        >
+          Filters
+        </Typography>
+        <DropdownFilter
+          placeholder="Select the Status"
+          value={status}
+          onChange={handleStatus}
+          lists={statusList}
+        />
+        <br />
+        <NumberFilter
+          placeholder="Select the Maximum Number of Participants"
+          value={filterParticipants}
+          onChange={handleFilterParticipants}
+        />
+        <br />
+        <DateFilter
+          placeholder="Select a Date Range"
```

Comment:
Is it possible to set the default status to be upcoming events in the next week? Maybe using "TODAY" and setting the end range to be TODAY + week or something? 

Not sure how difficult this is, I know you were having a time with the dumb date issues.

Reply:
setting default values for the filters would remove values from the results, and we want to show everything to user first and then allow them to filter from there. I think its better to allow the user to make this decision rather than making it for them if that makes sense

Reply:
The menu already opens to the current month, so I think that should be enough, a default status would cause the filter to enact right away - I think...

Reply:
I'm thinking its not exactly useful to look at events that have already occurred as a first look at all events.